docs/users/overview.adoc

```diff
@@ -10,10 +10,11 @@ You will quickly have an overview of the `bpmn-visualization` capabilities.
 === Usage
 
 `bpmn-visualization` can be directly used in HTML pages or integrated in Javascript and TypeScript applications. It is distributed as a https://npmjs.org/package/bpmn-visualization[npm package]
-in various bundle formats.
+in various bundle formats (IIFE, ESM & CommonJS).
 
-The API documentation is provided directly in the package thanks to the TypeScript declaration files that can be used in IDE for code assist.
-There is also a html version which can be previewed link:./api/index.html[here, window="_blank"].
+==== API
+The API documentation is provided directly in the package thanks to the TypeScript declaration files that can be used in IDE for code assist. +
+There is also an HTML version which can be previewed on link:./api/index.html[Github pages, window="_blank"].
```

Comment:
❓ As we add an API section, we should add another section for the _more technical details..._

Reply:
Done :)

Reply:
nice